Tasks/VsTest/vstest.ts

```diff
@@ -879,4 +894,10 @@ function responseContainsNoTests(filePath: string): Q.Promise<boolean> {
             return false;
         }
     });
+}
+
+function publishErrorToCi(errorCode, errorMessage) {
```

Comment:
initialize taskProps in local. not required global in this case. and use strongly typed properties

Reply:
strongly typed done

Reply:
done

---
In reply to: [136025114](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136025114) [](ancestors = 136025114)